src/core/public/chrome/ui/header/collapsible_nav_group_enabled.scss

```diff
@@ -1,33 +1,57 @@
 .context-nav-wrapper {
   border: none !important;
+  border-top-right-radius: $euiSizeL;
+  border-bottom-right-radius: $euiSizeL;
+  background-color: $euiColorLightShade;
 
   .nav-link-item {
-    padding: calc($euiSize / 4) $euiSize;
-    border-radius: $euiSize;
+    padding: $euiSizeS;
+    border-radius: $euiSizeS;
```

Comment:
Couldn't see the current selected app been highlighted in left nav, the reason is the left nav background color is the same as highlight nav item background color

<img width="676" alt="image" src="https://github.com/user-attachments/assets/cd263b6e-d88b-4d5e-8e36-6b5da4c84ef4">

Reply:
Sure, by using the new background color value, it should be able to tell the difference.